codefest/settings.py

```diff
@@ -167,16 +167,18 @@
 default_app = firebase_admin.initialize_app(cred)
 
 CORS_ORIGIN_WHITELIST = (
-    '*.codefest.tech',
-    '127.0.0.1:8080',
-    'localhost:8000',
-    '0.0.0.0:8080',
+    'https://*.codefest.tech',
+    'https://codefest.tech',
+    'http://http://127.0.0.1:8080',
+    'http://localhost:8000',
+    'http://0.0.0.0:8080',
 )
 CSRF_TRUSTED_ORIGINS = (
-    '*.codefest.tech',
-    '127.0.0.1:8080',
-    'localhost:8000',
-    '0.0.0.0:8080'
+    'https://*.codefest.tech',
```

Comment:
Check this out - https://docs.djangoproject.com/en/3.0/ref/settings/#csrf-trusted-origins